packages/templates/src/raw/baking-bread-guide/template.json

```diff
@@ -116,7 +116,7 @@
           "textAlign": "center",
           "padding": { "vertical": 0, "horizontal": 0, "locked": true },
           "type": "text",
-          "content": "<span style=\"font-weight: 900; color: #bcebee; letter-spacing: 0.4em\">BAK'N WITH EM</span>",
+          "content": "<span style=\"font-weight: 900; color: #bcebee; letter-spacing: 0.4em; text-transform: uppercase\">Bak'n with em</span>",
```

Comment:
`E` from em needs to be capital as it's short for Emma. So before text-transform it would be - Bak'n with Em